src/Model/TransportModel/tsp1.f90

```diff
@@ -144,7 +146,11 @@ subroutine tsp_cr(this, filename, id, modelname, macronym, indis)
     call budget_cr(this%budget, this%name)
     !
     ! -- create model packages
-    call this%create_tsp_packages(indis)
+    if (present(gwecommon)) then
```

Comment:
Maybe this is unavoidable, but seems unfortunate to have anything "gwe" in tsp.

Reply:
references to gwe-related code removed from `tsp1.f90` with 62ef0ba